subsys/partition_manager/CMakeLists.txt

```diff
@@ -130,6 +130,10 @@ if (CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_FLASH)
   ncs_add_partition_manager_config(pm.yml.modem_trace)
 endif()
 
+if (SB_CONFIG_VPR_LAUNCHER) # Currently doesn't work
```

Comment:
this will never be visible, the file here is something that runs per image, it does not run at a sysbuild context, those files are in cmake/sysbuild

Reply:
I guessed as much, so we need something that's visible here, how can we do that?

Reply:
I agree it should not be based on `SB_CONFIG_VPR_LAUNCHER`, we need to support both: builds with vpr_launcher and standalone ones.
Not sure about how PM works, so I'm most likely missing something - is it possible to use two separate yaml files for APP build and FLPR build?

Reply:
Ideally, you build everything at once. The point of PM is to have full knowledge of the system and partition things accordingly. In that case, you would use cpuapp as the top-level board instead of cpuflpr, so it would also solve the other problem I found. In that case you could always add the PM file for nrf54l15, and the user could optionally remove the flpr partition by setting its size to 0 in Kconfig.

But I understand that the current way is practical when developing for cpuflpr. We could maybe generalize the partition to something like "cpuapp_fw" since you would always need some FW for the appcore. I'm looking into making the necessary DTS overlays for building in this way.

Reply:
Changed it now